### PR TITLE
Update readme and add debug mode

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,45 @@ module.exports = function(config) {
...
+
+    asyncCall();
+  });
+});
+```
+
+#### Other Promise implementations
+
+By default `jasmine-stray-promises` hook into the `window.Promise` implementation, but custom ones can be added as long as they
+implement the thenable interface (i.e. has a "then" method on it's prototype).
```
Comment (kaicataldo) [inline README.md]: Nit: `has` -> `have`, `it's` -> `their`

**Review by kaicataldo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (weikinhuang) [reply README.md]: `have a "then" method on their prototype` sounds a little weird to me

**Review by nemtsov — CHANGES_REQUESTED** [review]
Needs a test?

File: README.md (hunk 1 above)
Comment (kaicataldo) [reply README.md]: Okay - just mismatching plural and singular, which reads weird to me. Gets the point across though. Maybe at least fix the `it's` => `its`?

**Comment by weikinhuang** [comment]
@nemtsov I don't think so, it's just more verbose output

File: README.md (hunk 1 above)
Comment (kaicataldo) [reply README.md]: Just pointing these things out since this is a public repo :)

**Comment by kaicataldo** [comment]
lgtm

**Comment by weikinhuang** [comment]
@nemtsov tests added

**Review by nemtsov — APPROVED** [review]

**Pull request merged**